Mathlib/Data/Countable/Basic.lean

```diff
@@ -53,24 +59,51 @@ section type
 
 variable {α : Type u} {β : Type v} {π : α → Type w}
 
-instance [Countable α] [Countable β] : Countable (Sum α β) := by
+instance [Countable α] [Countable β] : Countable (α ⊕ β) := by
   rcases exists_injective_nat α with ⟨f, hf⟩
   rcases exists_injective_nat β with ⟨g, hg⟩
   exact (Equiv.natSumNatEquivNat.injective.comp <| hf.sum_map hg).countable
 
+instance Sum.uncountable_inl [Uncountable α] : Uncountable (α ⊕ β) :=
+  inl_injective.uncountable
+
+instance Sum.uncountable_inr [Uncountable β] : Uncountable (α ⊕ β) :=
+  inr_injective.uncountable
+
 instance [Countable α] : Countable (Option α) :=
   Countable.of_equiv _ (Equiv.optionEquivSumPUnit.{_, 0} α).symm
 
+instance [Uncountable α] : Uncountable (Option α) :=
+  Injective.uncountable fun _ _ ↦ Option.some_inj.1
```

Comment:
```suggestion
  some_injective.uncountable
```

Reply:
That does not seem to work for me, I reverted this change!